docs/PayPalNativeCheckout/README.md

```diff
@@ -148,6 +149,67 @@ curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/order
 --data-raw ''
 ```
 
+## Billing Agreement
+
+### 1. Create Billing Agreement
+
+**Request**
+
+```bash
+curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/orders/' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <ACCESS_TOKEN>' \
+--data-raw '{
+  "description": "Billing Agreement",
+  "shipping_address":
+  {
+    "line1": "1350 North First Street",
+    "city": "San Jose",
+    "state": "CA",
+    "postal_code": "95112",
+    "country_code": "US",
+    "recipient_name": "John Doe"
+  },
+  "payer":
+  {
+    "payment_method": "PAYPAL"
+  },
+  "plan":
+  {
+    "type": "MERCHANT_INITIATED_BILLING",
+    "merchant_preferences":
+    {
+      "return_url": "https://example.com/return",
+      "cancel_url": "https://example.com/cancel",
+      "notify_url": "https://example.com/notify",
+      "accepted_pymt_type": "INSTANT",
+      "skip_shipping_address": false,
+      "immutable_shipping_address": true
+    }
+  }
+}'
+```
+
+**Response**
+
+```json
+{
+   "token_id": "<TOKEN>"
+}
+```
+
+### 2. Set Billing Agreement
+
+```kotlin
+paypalNativeClient.startCheckout(CreateOrder { createOrderActions ->
+   createOrderActions.setBillingAgreementId(billingAgreementId)
```

Comment:
Use the same variable name so it is clear that the merchant should pass this token response value into the `setBillingAgreementId` method . I also made this comment on the[ iOS PR](https://github.com/paypal/iOS-SDK/pull/92).
```suggestion
   createOrderActions.setBillingAgreementId("<BILLING_AGREEMENT_TOKEN>")
```